docs/api/search/search-success.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "SuccessResult",
```

Comment:
`SearchSuccessResult` would be a better name

Reply:
Makes sense. Made this change.